speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Recorder.cpp

```diff
@@ -16,10 +16,6 @@ Recorder::Recorder(const std::string& p_dirName, const std::string& p_fileName,
 {
     // create directory if it doesn't exist
     std::string dirPath = "..\\test_data\\" + p_dirName;
-    if (!GfDirExists(dirPath.c_str()))
-    {
-        GfDirCreate(dirPath.c_str());
-    }
```

Comment:
It still needs to create the directory right? It just needs a different location

Reply:
No it is created by cmake now

Reply:
Also when you install it?

Reply:
yes, but it has been changed now